### DISCUSSION TITLE
Abandoned? Worth starting now? Alternatives

### DISCUSSION THREAD

**Discussion by Dud3z** [FireMarshmellow/Spotlight_Storage#67] [General]
Hi, is the project abandoned? Seems like dev has no interest in this Project anymore. Are there any Alternatives except Homebox Partsdb. i need something very simple. Name Quantity Location Picture.

**Comment by FireMarshmellow** [comment]
No there's a team of 2 working on updating as we speak, we just develop on a different fork. https://github.com/JustReit/M.I.M.O.S.A

**Reply by Dud3z** [reply]
okay, i get it. Why dont you mark it as archived and write into the readme that it is developed on another fork? Its not that easy to find aktive forks for beginners =)

**Reply by FireMarshmellow** [reply]
This is still the main one, and it will be updated with all the latest features when we are ready
